lib/xclarity_client/services/update_policy_management.rb

```diff
@@ -0,0 +1,52 @@
+require 'json'
+
+module XClarityClient
+  class UpdatePolicyManagement < Services::XClarityService
+    manages_endpoint UpdatePolicy
+
+    def get_applicable_firmware()
+      get_update_policy("applicableFirmware", {})
```

Comment:
@umrikar talking with @CharlleDaniel we think that would be good to add these `uri` strings to the endpoint class `UpdatePolicy` to look like the [`User`](https://github.com/lenovo/xclarity_client/blob/master/lib/xclarity_client/endpoints/user.rb) endpoint `SUB_URIS` constants

Reply:
done